crates/ursa-gateway/src/resolver/mod.rs

```diff
@@ -23,6 +23,12 @@ pub struct Resolver {
     client: Client,
 }
 
+#[derive(Debug)]
+pub struct Response {
```

Comment:
Maybe `CachNodeResponse` for clarity, as `Response` is a bit vague.

Reply:
That's pretty verbose. How about `NodeResponse`? :)